model_analyzer/record/record_aggregator.py

```diff
@@ -160,3 +220,23 @@ def aggregate(self, record_types=None, reduce_func=max):
                 values.append(record.value())
             aggregated_records[record_type] = reduce_func(values)
         return aggregated_records
+
+    def get_records(self):
```

Comment:
can we add a parameter/option to get all records of a specific type/ list of types?
```
def get_records(self, record_types):
     # Construct a subset of self._records
```

I'm not sure if this is a workable idea as the calling code needs to be aware of the types.

Reply:
Yeah, I guess we can add this later if we found a place where this was required.